.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -2208,6 +2209,708 @@ public static class OpenAIModerationsModelFactory {
         public static ModerationResultCollection ModerationResultCollection(string id = null, string model = null, IEnumerable<ModerationResult> items = null);
     }
 }
+namespace OpenAI.RealtimeConversation {
```

Comment:
Should the namespace just be "Realtime"?

Reply:
let's defer and discuss with the bigger design push -- this is a 1:1 with the current name of the client, which we very may well want to change, too.